source/common/modules/markdown-editor/hooks/autocomplete.ts

```diff
@@ -473,6 +473,10 @@ function hintFunction (cm: CodeMirror.Editor, opt: CodeMirror.ShowHintOptions):
     if (autocompleteStart === null) {
       throw new Error('Could not autocomplete: autocompleteStart was null')
     }
+    if (currentDatabase == null) {
+      // We don't know where to get completion data from, so we can't do anything.
```

Comment:
Here we should throw an error analogous to above because this really shouldn't happen and would indicate a bug